### PR TITLE
100% code coverage

### LINKED ISSUE #1003: 100% code coverage
Not important, but would be nice.

We can use `/* istanbul ignore next: Some comment */` type comments where it doesn't make sense to test a code path.

### PR DESCRIPTION
**Pull request by fisker**
Fixes #1003, 100% code coverage

### CODE REVIEW

File: rules/prefer-query-selector.js [hunk 1]
```diff
@@ -106,9 +106,6 @@ const create = context => {
 		[selector](node) {
 			const method = node.callee.property.name;
 			const preferedSelector = forbiddenIdentifierNames.get(method);
```
Comment (sindresorhus) [inline rules/prefer-query-selector.js:108]: Unrelated, but I noticed a typo here.
```suggestion
			const preferredSelector = forbiddenIdentifierNames.get(method);
```

File: rules/custom-error-definition.js [hunk 2]
```diff
@@ -161,14 +161,6 @@ const customErrorDefinition = (context, node) => {
 };
 
 const customErrorExport = (context, node) => {
-	if (!node.left.object || node.left.object.name !== 'exports') {
-		return;
-	}
-
-	if (!node.left.property) {
-		return;
-	}
```
Comment (fisker) [inline rules/custom-error-definition.js:170]: Moved this part to selector

File: rules/prefer-query-selector.js [hunk 3]
```diff
@@ -88,39 +88,36 @@ const hasValue = node => {
...
 
 const create = context => {
 	return {
 		[selector](node) {
 			const method = node.callee.property.name;
-			const preferedSelector = forbiddenIdentifierNames.get(method);
-			if (!preferedSelector) {
-				return;
-			}
+			const preferredSelector = forbiddenIdentifierNames.get(method);
```
Comment (fisker) [inline rules/prefer-query-selector.js:108]: Selector already list the methods, so we always have `preferredSelector`

File: rules/consistent-function-scoping.js [hunk 4]
```diff
@@ -119,9 +121,6 @@ function checkNode(node, scopeManager) {
 	}
 
 	let parentNode = node.parent;
-	if (!parentNode) {
```
Comment (fisker) [inline rules/consistent-function-scoping.js:122]: Function parent can't be null

**Comment by sindresorhus** [comment]
Nice! 🎉

**Pull request merged**